client/app/redux/actions/FilterActions.js

```diff
@@ -35,21 +35,20 @@ export function getFilters(expoToken) {
     }
 }
 
-export function addFilter(menuItemID, menuItemName) {
+export function addFilter(allergen) {
```

Comment:
semantically, if this is an `addFilter` method, the argument could be `filter` and not `allergen` for consistency?